src/system.jl

```diff
@@ -1,143 +1,103 @@
 """
-    ServicesTimeSeries(reg, spin, sup_on, sup_off)
-
-Type defining services time series.  Fields are KeyedArray where the keys are generator names
-x datetimes.  Only generators that provide each service are included in the array.
-"""
-struct ServicesTimeSeries
-    reg::KeyedArray{Float64}
-    spin::KeyedArray{Float64}
-    sup_on::KeyedArray{Float64}
-    sup_off::KeyedArray{Float64, 2}
-end
-
-"""
-    Zone(number, reg, spin, sup_on, sup_off)
+    $TYPEDEF
 
 Type defining a market zone.  The `Zone` is identified by a number.  The other fields contain
 the service requirements for the zone.
+
+Fields:
+$TYPEDFIELDS
 """
 struct Zone
+    "Zone number"
     number::Int64
+    "Zonal regulation requirement (MWs)"
```

Comment:
Are these `MW` or `MWh`? 

Also we should not pluralise units

```suggestion
    "Zonal regulation requirement (MW)"
```
or
```suggestion
    "Zonal regulation requirement (MWh)"
```

Reply:
We're generally lax with MW vs MWh because we model things hourly which means that if `p = 100` that could either mean that the generator has a power output of 100 MW for one hour, or that the generator produced a total of 100 MWh across that hour (same thing applies virtually everywhere). I think it's generally easier to see it as MW and follow the logic that we're assuming things are static within a single time period (which in our case is one hour)

Reply:
I've changed it to use MW